### PR TITLE
Bump Rails version

### PR DESCRIPTION
**Pull request by patbenatar**
@taboularasa

### CODE REVIEW

File: lib/suspenders/version.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module Suspenders
-  RAILS_VERSION = "4.2.1"
+  RAILS_VERSION = "4.2.3"
```
Comment (taboularasa) [inline lib/suspenders/version.rb:2]: we want to always use the latest stable so maybe this value should be dynamic. how? `gem install rails --explain` ? probably a better way?

Comment (patbenatar) [inline lib/suspenders/version.rb:2]: @taboularasa Agreed. I vote to split this out as separate issue for enhancement?

Comment (taboularasa) [inline lib/suspenders/version.rb:2]: @patbenatar ok

Comment (patbenatar) [inline lib/suspenders/version.rb:2]: #51 ... I can has squirrel?

**Comment by taboularasa** [comment]
:shipit:

**Pull request merged**